client/app/hearings/utils.js

```diff
@@ -920,4 +920,17 @@ export const getTimezoneAbbreviation = (timezone) => {
   return moment.tz('00:00', 'HH:mm', timezone).format('z');
 };
 
+export const formatNotificationLabel = (hearing, virtual, appellantTitle) => {
+  const poaLabel = virtual ? ', POA,' : ' and POA';
+  const recipientLabel = hearing?.representative ? `${appellantTitle}${poaLabel}` : `${appellantTitle}`;
+
+  if (virtual) {
+    return `When you schedule the hearing, the ${recipientLabel} and` +
```

Comment:
```suggestion
    return `When you schedule the hearing, the ${recipientLabel} and ` +
```
I think a space is needed here.